### ISSUE TITLE
MaxResourceVariance clarification

### ISSUE THREAD

**Issue by an-accident** [Sh0wdown/UnofficialCrusaderPatch#454]
From the commentary provided in .aic it seems like you wasn't sure what MaxResourceVariance  really does. I've done some testing. It looks like this is equivalent of MaxEquipment used for equipment that is not set in BlacksmithSetting, FletcherSetting and PoleturnerSetting.
Example: Sherif produces maces, crossbows and pikes. So when he gets into possesion of some swords, bows or spears he sells them as soon as they pass MaxResourceVariance as opposed to MaxEquipment used for other weapons.
TL;DR: in AI Personality:
MaxEquipment  - Applies to weapons and armour **produced** by AI
MaxResourceVariance  - Applies to weapons and armour **which are not produced** by AI

**Comment by ilosevsrat** [comment]
Thnx alot

**Comment by Russianstory** [comment]
My friend, you touched on an important topic. Indeed, the Bot changes everything that he does not need ... he immediately sells everything that was not mortgaged from him by the creator. In addition to gold))). When I first played, I suffered greatly from this. You help the Snake, throw maces and he sells them. Is he a fool? But no ... just conceived. But it would be very cool if the AI ​​could hire soldiers who are not provided for it. BUT under two conditions
1) This is done by a live player (this is done artificially)
2) To hire swordsmen, arbalester, macemen, pikemen, it is necessary to send both weapons and armor.
-
Дружище ты затронул важную тему. Да и правда Бот меняет всё то, что ему не нужно...он продаёт сразу всё что не заложено у него создателем. Кроме золота ))). Я поначалу когда играл сильно страдал от этого. Ты помогаешь Змее, кидаешь булавы а он их продаёт. Дурак что-ли? А нет...просто так задумано. Но было бы очень здорово если бы ИИ мог нанимать солдат тех которые у него не предусмотрены. НО при двух условиях
1) Это делается живым игроком (это делается искусственно)
2) Для найма мечников, арбалетчиков, булавщиков, пикинёров необходимо отправить и оружие и доспехи.

**Comment by ByBurton** [comment]
Uhm, what?
MaxResourceVariance functions just as is written in the comment.
The AI checks its stone for example. If MaxStones is set to let's say 50 and MaxResourceVariance is set to 4, then the AI will only start selling stone above the value 54, but back to 50 and not 54.
Same for equipment. If the MaxEquipment is set to 5 and MaxResourceVariance to 4 again, then the AI will only start selling stuff above 9 and back to 5. (Although I noticed the AI sometimes not selling equipment).
The Sheriff is selling swords because Swords are listed on his list of goods he immediately sells.
So you are wrong.
